app/graphql/types/user_error.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  class UserError < BaseObject
```

Comment:
I find the class name a bit misleading, only after reading the description it was clear that the error is not related to `User` model. How about ValidationError, AttributeError or something similar?